src/Pages/Withdrawals.razor

```diff
@@ -165,23 +166,31 @@
                             }
                         </DisplayTemplate>
                         <EditTemplate>
-                            @if (!_isAmountDisabled)
-                            {
-                                <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args, _isAmountDisabled)">
-                                    <NumericPicker TValue="decimal" Disabled="_isAmountDisabled" Value="@((decimal)context.CellValue)" ValueChanged="(value) => { context.CellValue = value; }" CurrencySymbol="₿ " Max="@(_maxWithdrawal)" Min="_minWithdrawal" Decimals="8">
-                                        <Feedback>
-                                            <ValidationError />
-                                        </Feedback>
-                                    </NumericPicker>
-                                    <FieldHelp>
-                                        @($"Current amount: {Math.Round(PriceConversionService.BtcToUsdConversion((decimal)context.CellValue, _btcPrice), 2)} USD")
-                                    </FieldHelp>
+                                <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args, false)">
+                                    <NumericPicker TValue="decimal" @bind-Value="@_amount" Step="0.00001m" CurrencySymbol="₿ " Max="@(_maxWithdrawal)" Min="_minWithdrawal" Decimals="8" />
+                                     <FieldHelp>
+                                       @if (Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT < _amount)
+                                       {
+                                           <span class="text-danger">@($"Maximum amount {Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT} BTC.")</span>
+                                       }
+                                       else
+                                       {
+                                           @($"Amount in BTC. Minimum {_minimumChannelCapacity:f8}. Current amount: {Math.Round(PriceConversionService.SatToUsdConversion(new Money(_amount, MoneyUnit.BTC).Satoshi, _btcPrice), 2)} USD")
```

Comment:
`_minimumChannelCapacity` is a channel related variable. This should be something related with withdrawal.